.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
Hmm, it works on my machine, but these dont matter so I'll remove

Reply:
https://code.visualstudio.com/docs/languages/json#_json-with-comments

apparently vscode has some special type of json that allows comments, so we can leave it since its in the .vscode folder anyway?

Reply:
did not know that, sure then we can leave it